oauth_dropins/indieauth.py

```diff
@@ -54,6 +55,39 @@ def discover_authorization_endpoint(me, resp=None):
     return auth_endpoint
   return INDIEAUTH_URL
 
+def discover_token_endpoint(me, resp=None):
```

Comment:
this is the same as `discover_authorization_endpoint` except for the rel value, could you unify them?